src/browser-frontend/views/browser/window.jsx

```diff
@@ -17,9 +17,8 @@ import Mousetrap from 'mousetrap';
 
 import { client } from '../../global';
 import PagesModelActions from '../../actions/pages-model-actions';
-// import * as SharedPropTypes from '../../model/shared-prop-types';
-// import SharedActions from '../../../shared/actions/shared-actions';
-// import * as UIPagesSelectors from '../../selectors/ui-pages-selectors';
```

Comment:
Commented out parts in this PR are part of the backouts performed in f148577c0ed17778d4f7432dfca4c4094a1d8ed7 and d5bbd159274ad0ab54243fc84bdb90204219e942